examples/advance_music_cog.py

```diff
@@ -469,6 +469,32 @@ async def previous(self, ctx, index: int = None):
         if previous_player := await self.MusicManager.previous(ctx, index):
             await ctx.send(f"Rewinding from {previous_player[0].title}")
 
+    # Spotify song from user
+    @commands.command()
+    async def play_user_spotify(self, ctx, user: discord.Member = None):
+        user = user if user else ctx.guild.get_member(ctx.author.id)
```

Comment:
Please rename user to member for clarity.
Unnecessary to get the author id from the guild. Perhaps do this instead:
```py
member = member if member else ctx.author
```